pycode/memilio-epidata/memilio/epidata_test/test_epidata_defaultDict.py

```diff
@@ -20,53 +20,64 @@
+    }
     test_dict_state = {
-        'Schleswig-Holstein': 1}, {
-        'Hamburg': 2}, {
-        'Niedersachsen': 3}, {
-        'Bremen': 4}, {
-        'Nordrhein-Westfalen': 5}, {
-        'Hessen': 6}, {
-        'Rheinland-Pfalz': 7}, {
-        'Baden-Württemberg': 8}, {
-        'Bayern': 9}, {
-        'Saarland': 10}, {
-        'Berlin': 11}, {
-        'Brandenburg': 12}, {
-        'Mecklenburg-Vorpommern': 13}, {
-        'Sachsen': 14}, {
-        'Sachsen-Anhalt': 15}, {
-        'Thüringen': 16}
+        'Schleswig-Holstein': 1,
+        'Hamburg': 2,
+        'Niedersachsen': 3,
+        'Bremen': 4,
+        'Nordrhein-Westfalen': 5,
+        'Hessen': 6,
+        'Rheinland-Pfalz': 7,
+        'Baden-Württemberg': 8,
+        'Bayern': 9,
+        'Saarland': 10,
+        'Berlin': 11,
+        'Brandenburg': 12,
+        'Mecklenburg-Vorpommern': 13,
+        'Sachsen': 14,
+        'Sachsen-Anhalt': 15,
+        'Thüringen': 16
+    }
     test_dict_dd = {
-        date.today(): 'end_date'}, {
-        date(2020, 4, 24): 'start_date'}, {
-        'json_timeasstring': 'file_format'}
-    test_values_notin_dd = ('no_raw', 'impute_dates', 'split_berlin', 'update_data', 'make_plot')
-
-    def setUp(self):
-        self.setUpPyfakefs()
+        date.today(): 'end_date',
+        date(2020, 4, 24): 'start_date',
+        'json_timeasstring': 'file_format'
+    }
+    test_values_notin_dd = ('no_raw', 'impute_dates', 'split_berlin', 
+                            'update_data', 'make_plot')
 
     def test_invert_dict(self):
         inv_county = dd.invert_dict(dd.County)
-        for item in self.test_dict_county:
-            self.assertDictContainsSubset(item, inv_county)
+        # check that test_dict_county is a subset of inv_county
+        for county in self.test_dict_county.keys():
+            self.assertIn(county, inv_county.keys())
+            self.assertEqual(self.test_dict_county[county], 
+                             inv_county[county])
+        # check inverted state dictionary complete
         inv_state = dd.invert_dict(dd.State)
-        for item in self.test_dict_state:
-            self.assertDictContainsSubset(item, inv_state)
+        self.assertDictEqual(self.test_dict_state, inv_state)
+        # check that test_dict_dd is a subset of inv_dd
         inv_dd = dd.invert_dict(dd.defaultDict)
-        for item in self.test_dict_dd:
-            self.assertDictContainsSubset(item,inv_dd)
+        for defaultvalue in self.test_dict_dd.keys():
+            self.assertIn(defaultvalue, inv_dd.keys())
+            self.assertEqual(self.test_dict_dd[defaultvalue], 
+                             inv_dd[defaultvalue])
         for value in self.test_values_notin_dd:
             self.assertNotIn(value, inv_dd.values())
 
+    def test_uniqueness(self):
+        # tests whether the names of counties and states are unique
+        self.assertEqual(dd.invert_dict(dd.invert_dict(dd.County)), dd.County)
```

Comment:
This does not fail even if the order in the dictionary is changed. Otherwise this would also be an issue for the test of the state dictionary. The loop is only done if only a part  of the entries is checked.